e2e/provision/gce_install_sandbox.sh

```diff
@@ -71,10 +74,24 @@ else
         ansible-playbook -vvv -i ~/nephio.yaml playbooks/cluster.yml
     fi
 
-    # Deploy free5GC workload
-    if [[ ${DEBUG:-false} != "true" ]]; then
-        ansible-playbook -i ~/nephio.yaml playbooks/free5gc.yml
-    else
-        ansible-playbook -vvv -i ~/nephio.yaml playbooks/free5gc.yml
-    fi
+    # Put this in the ubuntu dir and make it accessible to world
+    mkdir "$HOME/.kube" && chmod 755 "$HOME/.kube"
+    sudo cp /root/.kube/config "$HOME/.kube"
+    sudo chown $USER:$USER "$HOME/.kube/config"
+    chmod 644 "$HOME/.kube/config"
+
+    # I don't know how to make ansible do what I want, this is what I want
+    mkdir /tmp/mgmt-repo
+    /usr/local/bin/kpt pkg get --for-deployment https://github.com/nephio-project/nephio-example-packages.git/repository@repository/v2 /tmp/mgmt-repo/mgmt
+    # sudo because docker
+    sudo /usr/local/bin/kpt fn render /tmp/mgmt-repo/mgmt
+    /usr/local/bin/kpt live init /tmp/mgmt-repo/mgmt
+    /usr/local/bin/kpt live apply /tmp/mgmt-repo/mgmt
+
+    mkdir /tmp/mgmt-rootsync
+    /usr/local/bin/kpt pkg get --for-deployment https://github.com/nephio-project/nephio-example-packages.git/rootsync@rootsync/v2 /tmp/mgmt-rootsync/mgmt
+    # sudo because docker
+    sudo /usr/local/bin/kpt fn render /tmp/mgmt-rootsync/mgmt
+    /usr/local/bin/kpt live init /tmp/mgmt-rootsync/mgmt
+    /usr/local/bin/kpt live apply /tmp/mgmt-rootsync/mgmt
```

Comment:
Those instructions seems to belong to the workload process, right?

So if that's the case the can be included as a pre_task [here](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/free5gc.yml#L14-L21)

```yaml
    - role: kpt
      repo_uri: https://github.com/nephio-project/nephio-example-packages
      local_dest_directory: /tmp/mgmt-repo/mgmt
      pkg: repository
      version: repository/v2
      context: kind-kind
      namespaces:
        - default
```
That [role](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/roles/kpt/tasks/main.yml) will execute the following instructions on the package:

* creates the base folder if doesn't exists (`/tmp/mgmt-repo/mgmt)
* kpt pkg get (If it hasn't done)
* kpt pkg tree
* kpt fn render
* kpt pkg diff
* kpt live init
* kpt live apply (5 attempts)

Reply:
thanks, I tried that but it didn't work exactly like I wanted:

1. We probably need to update that to use `--for-deployment`
2. I need the package name to be "mgmt", so I need the actual kpt command that runs to be the one in the script - I think it did something a little different, I ended up with /tmp/mgmt-repo/mgmt/repository instead of /tmp/mgmt-repo/mgmt. The idea is that we rename the package during the "get" - that rename actually is meaningful, it triggers some automation.